lib/utils.js

```diff
@@ -50,7 +50,8 @@ function isDomElement(o) {
 }
 
 function isSpecialClick(event) {
-  return event.altKey || event.ctrlKey || event.metaKey || event.shiftKey;
+  let isMiddleClick = (event.button === 1);
```

Comment:
I wonder, I feel that's a habit to put this expression in parenthesis but it really look weird to me all the time. There's really no need for it and here it does not make the code clearer (again, habits).

But more importantly, why not have those parenthesis on the next line also then?

I thought we had http://eslint.org/docs/rules/no-extra-parens but it's not triggerint here, weird